ui/cryptomaterial/dropdown.go

```diff
@@ -134,41 +150,47 @@ func (d *DropDown) Changed() bool {
 	return false
 }
 
-func (d *DropDown) layoutActiveIcon(gtx layout.Context, index int) D {
+func (d *DropDown) layoutActiveIcon(gtx C, index int) D {
 	var icon *Icon
 	if !d.isOpen {
 		icon = NewIcon(d.dropdownIcon)
+		icon.Color = d.theme.Color.Gray1
 	} else if index == d.selectedIndex {
 		icon = NewIcon(d.navigationIcon)
+		if d.NavigationIconColor != nil {
+			icon.Color = *d.NavigationIconColor
+		} else {
+			icon.Color = d.theme.Color.Gray1
+		}
 	}
```

Comment:
Set the color outside this if/else block to simplify things:
```
if d.isOpen || d.ClosedIconColor == nil {
	icon.Color = d.theme.Color.Gray1
} else {
	icon.Color = *d.ClosedIconColor
}
```

I assumed that NavigationIconColor actually means ClosedIconColor, although I don't even think "Closed Icon" makes sense.

Reply:
Will find a suitable name for it now that you’ve mentioned renaming the icon.